rules/Naming/ExpectedNameResolver/InflectorSingularResolver.php

```diff
@@ -5,25 +5,19 @@
 namespace Rector\Naming\ExpectedNameResolver;
 
 use Nette\Utils\Strings;
+use PHPStan\Reflection\ReflectionProvider;
 use Rector\Core\Util\StringUtils;
 
 /**
  * @see \Rector\Core\Tests\Naming\ExpectedNameResolver\InflectorSingularResolverTest
  */
 final class InflectorSingularResolver
 {
-    /**
-     * @var array<string, string>
-     */
-    private const SINGULARIZE_MAP = [
-        'news' => 'new',
-    ];
-
     /**
      * @var string
      * @see https://regex101.com/r/lbQaGC/3
      */
-    private const CAMELCASE_REGEX = '#(?<camelcase>([a-z\d]+|[A-Z\d]{1,}[a-z\d]+|_))#';
+    private const CAMELCASE_REGEX = '#([a-z\d]+|[A-Z\d]{1,}[a-z\d]+|_)#';
```

Comment:
could you update `@see https://regex101.com ` uri?

Reply:
> could you update `@see https://regex101.com ` uri?

sure. do we agree with the [BC related breakages](https://github.com/rectorphp/rector-src/actions/runs/6638636120/job/18035385925?pr=5200)?

Reply:
I will let @TomasVotruba decide, if the changes are too much, I think re-use doctrine/inflector seems the way to go

Reply:
I created a PR to revert doctrine/inflector removal https://github.com/rectorphp/rector-src/pull/5201 that I can let @TomasVotruba decide